### PR TITLE
New cheat sheet for GNU Screen

### PR DESCRIPTION
**Pull request by dnmfarrell**

### CODE REVIEW

**Comment by zekiel** [comment]
@dnmfarrell  :D Will review!

File: lib/DDG/Goodie/GnuScreenCheatSheet.pm [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    'quick reference screen',
+    'help screen',
+);
+
+attribution github  => ["dnmfarrell",            "David Farrell"],
+            twitter => ["perltricks",           "David Farrell"],
+            web     => ["http://perltricks.com", "David Farrell"];
+
+my $HTML = share("tmux_cheat_sheet.html")->slurp(iomode => '<:encoding(UTF-8)');
+my $TEXT= share("tmux_cheat_sheet.txt")->slurp(iomode => '<:encoding(UTF-8)');
```
Comment (jagtalon) [inline lib/DDG/Goodie/GnuScreenCheatSheet.pm]: Should access gnuscreen_cheat_sheet.html and gnuscreen_cheat_sheet.txt instead of tmux

**Comment by jagtalon** [comment]
@dnmfarrell the folder name in the `share` directory should be `gnu_screen_cheat_sheet` instead of `gnuscreen_cheat_sheet`. Running `prove -Ilib t/GnuScreenCheatSheet.t` gives me:

```
t/GnuScreenCheatSheet.t .. Can't read /home/codio/workspace/zeroclickinfo-goodies/share/goodie/gnu_screen_cheat_sheet/tmux_cheat_sheet.htm
l: No such file or directory at /home/codio/workspace/zeroclickinfo-goodies/lib/DDG/Goodie/GnuScreenCheatSheet.pm line 40.                
Compilation failed in require at t/GnuScreenCheatSheet.t line 19.                                                                         
t/GnuScreenCheatSheet.t .. Dubious, test returned 2 (wstat 512, 0x200)                                                                    
No subtests run                                                                                                                           
                                                                                                                                          
Test Summary Report                                                                                                                       
-------------------                                                                                                                       
t/GnuScreenCheatSheet.t (Wstat: 512 Tests: 0 Failed: 0)                                                                                   
  Non-zero exit status: 2                                                                                                                 
  Parse errors: No plan found in TAP output                                                                                               
Files=1, Tests=0,  0 wallclock secs ( 0.03 usr  0.08 sys +  0.14 cusr  0.04 csys =  0.29 CPU)                                             
Result: FAIL                                                                                            
```

Which basically says, I didn't find the share directory.

**Comment by jagtalon** [comment]
@dnmfarrell looks like I'm only seeing one column. Could you make it two like Regex (https://duckduckgo.com/?q=regexp+cheat+sheet&ia=answer) and Vim (https://duckduckgo.com/?q=vim+cheat+sheet&ia=answer)?

screen shot 2015-04-04 at 2 52 28 pm

**Comment by dnmfarrell** [comment]
Thanks jagtalon. Let me start a new request with the corrections

**Pull request closed without merging**